src/components/menu-mobile/Menu.vue

```diff
@@ -1,40 +1,129 @@
 <template>
-  <div class="mew-component--wallet-menu-mobile">
-    <v-overlay :value="value" opacity="0.95">
-      <div style="overflow-y: auto">
-        <v-btn class="exit" x-large icon @click="$emit('input')">
-          <v-icon>mdi-window-close</v-icon>
-        </v-btn>
+  <div class="mew-component--landing-page-menu-mobile">
```

Comment:
1. Can you please move this component under views/components-default
2. Rename component as TheDefaultMobileNavigation and make sure the imports have the same name

Reply:
Fixed

Reply:
Done